NaviGator/mission_control/navigator_alarm/navigator_alarm_handlers/thruster_fault.py

```diff
@@ -1,6 +1,8 @@
+from __future__ import absolute_import
 from ros_alarms import HandlerBase, AlarmBroadcaster
 from roboteq_msgs.msg import Status
 import rospy
+import six
```

Comment:
Another thing that I didn't explain super well is that we are avoiding `six`, and trying to use vanilla Python changes. `import six` imports can be removed.